packages/docs-reanimated/src/examples/CustomEnteringExiting.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { View, Button, StyleSheet, SafeAreaView } from 'react-native';
+import Animated, { withTiming, withDelay } from 'react-native-reanimated';
+
+const WIDTH = 200;
+
+export default function EnteringExample() {
+  const [show, setShow] = useState(false);
+
+  const customEntering = (targetValues) => {
```

Comment:
I believe both `customEntering` and `customExiting` should be defined outside of the component body to avoid unnecessary calculations when component rerenders